directory-connector/src/com/untangle/app/directory_connector/DirectoryConnectorApp.java

```diff
@@ -244,6 +250,20 @@ public void setSettings(final DirectoryConnectorSettings newSettings)
         this.reconfigure();
     }
 
+     /**
+     * Encrypt the password for AD Servers
+     *
+     * @param settings
+     *      Settings to convert.
+     */
+    private void setEncryptedPassword( DirectoryConnectorSettings settings ) {
+        ActiveDirectorySettings adSettings = settings.getActiveDirectorySettings();
+        for(ActiveDirectoryServer server : adSettings.getServers()) {
+            if(!StringUtils.isBlank(server.getSuperuserPass()))
+                server.setEncrSupUserPass(null);
```

Comment:
Use the overloaded method `setEncrSupUserPass()` to set encrypted user password.